set.mm

```diff
@@ -1551,7 +1551,7 @@ between the hypotheses and the assertion (conclusion), abbreviating the
     $( Inference associated with ~ pm2.86 .  (Contributed by NM, 5-Aug-1993.)
        (Proof shortened by Wolf Lammen, 3-Apr-2013.) $)
```

Comment:
Could you add to the comment of pm2.86i:
> An equally short proof is possible from ~ ax-1 and ~ syl11 .

so that we do not lose this information?

Reply:
Both proofs represent the sequence ax-1, syl, com12.  The old one pairs the two last, the new one the first two steps.  So which one is to be preferred? Asking me, the former looks like a technical shortcut, the latter has, from a pedagogical point of view, slightly more relevance.  I can create an OLD version of course, but is it really worth it?

Reply:
It's not worth creating an OLD or ALT version. I thought one sentence in the comment was appropriate. But if you think it's not, then as you prefer.

Reply:
This opens a can of worms: Keep a myriad of variants in the theorem comment, for what reason?  To not get any information lost?  If the idea is not valuable enough to make it to an ALT version, I'd rather see it discarded, as was the case with numerous alternative proof ideas I threw away in a shortening process.  I am not really convinced.  A frequent remedy to that I occasionally encounter. is: use the git history commands.

Reply:
We do not have to be systematic in everything we do. I wouldn't have added this comment if I'd discovered this alternate proof. It is the action of removing it which made me think it could be kept in a comment at a very low cost. But that's not very important indeed.

I merged the PR anyway.